forge/src/main/java/org/spongepowered/forge/hook/ForgeEventHooks.java

```diff
@@ -25,19 +25,48 @@
 package org.spongepowered.forge.hook;
 
 import net.minecraft.server.level.ServerLevel;
+import net.minecraft.server.level.ServerPlayer;
 import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.entity.vehicle.AbstractMinecartContainer;
 import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.phys.BlockHitResult;
 import net.minecraftforge.common.ForgeHooks;
+import net.minecraftforge.common.MinecraftForge;
 import net.minecraftforge.event.ForgeEventFactory;
 import net.minecraftforge.event.entity.player.CriticalHitEvent;
+import net.minecraftforge.event.entity.player.PlayerInteractEvent;
+import org.spongepowered.api.event.CauseStackManager;
+import org.spongepowered.api.event.block.InteractBlockEvent;
 import org.spongepowered.api.event.entity.ChangeEntityWorldEvent;
+import org.spongepowered.api.util.Tuple;
+import org.spongepowered.common.event.tracking.PhaseTracker;
 import org.spongepowered.common.hooks.EventHooks;
+import org.spongepowered.forge.launch.bridge.event.ForgeEventBridge_Forge;
+import org.spongepowered.forge.launch.event.ForgeEventManager;
 
 public final class ForgeEventHooks implements EventHooks {
 
+    @Override
+    public Tuple<InteractBlockEvent.Secondary, InteractionResult> callInteractBlockEventSecondary(
```

Comment:
As discussed on discord, this hook can be removed if we add the "cancellation result" to the API.